src/couch/src/couch_task_status.erl

```diff
@@ -12,7 +12,6 @@
 
 -module(couch_task_status).
 -behaviour(gen_server).
--vsn(1).
```

Comment:
?

Reply:
Why have it? It's just noise.

Reply:
We removed the vsn attribute in a separate PR: https://github.com/apache/couchdb/pull/4853